packages/veritone-json-schemas/.babelrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Every package in veritone-sdk can use a different .babelrc, since it's probably too onerous to force all of it to use the same babel version. Frontend projects tend to me much more current than backend in terms of ES stage support.